src/Fantomas/CodePrinter.fs

```diff
@@ -2453,7 +2453,12 @@ and genExpr astContext synExpr ctx =
 
             expressionFitsOnRestOfLine (short +> idx) (long +> idx)
         | DotIndexedGet (objectExpr, indexArgs) ->
-            addParenIfAutoNln objectExpr (genExpr astContext)
+            let isParen =
```

Comment:
This is quite confusing, `isParen` is false when the expression is `SynExpr.Paren`.

Reply:
Done